language/tools/move-package/tests/test_sources/external_overlap_fail/Move.resolved

```diff
@@ -0,0 +1,3 @@
+Failed to resolve dependencies for package 'Root': Adding dependencies from ../resolvers/successful.sh for dependency 'A' in 'Root': Conflicting dependencies found:
+ADep = { local = "deps_only/ADep" }
+ADep = { local = "./deps_only/ADep" } # Resolved by ../resolvers/successful.sh
```

Comment:
We should probably bolster this error now -- the issue is not with `ADep` but the fact that its own dependencies differ, so we should share that information to developers.